app_tests/integration_tests/llm/utils.py

```diff
@@ -129,7 +129,7 @@ def find_available_port():
         return port
 
 
-def wait_for_server(url, timeout=10):
+def wait_for_server(url, timeout=60):
```

Comment:
FYI, unit tests also failed to start the server after some IREE changes, noted on https://github.com/nod-ai/shark-ai/pull/722#issuecomment-2557873661.